<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="partOneWidget">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>80</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Первая часть</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>418</y>
          <width>582</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonLedON">
           <property name="text">
            <string>Включить светодиод</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Bebas Neue Cyrillic</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Светодиод выключен</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButtonLedOFF">
           <property name="text">
            <string>Выключить светодиод</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButtonSecondPart">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>20</y>
          <width>90</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Вторая часть</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Подключение COM-порта</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>571</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Communication</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>551</width>
           <height>26</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="Connection">
            <property name="contextMenuPolicy">
             <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ComPort"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonConnectCOMport">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>381</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClearTextEdit">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="partTwoWidget">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>80</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Вторая часть</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonFirstPart">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>20</y>
          <width>90</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Первая часть</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>310</y>
          <width>196</width>
          <height>40</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QSlider" name="horizontalSliderPWM">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelPWM">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_pwmLED">
           <property name="text">
            <string>Уровень яркости</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>420</y>
          <width>581</width>
          <height>119</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelStateLed2">
           <property name="font">
            <font>
             <family>Bebas Neue Cyrillic</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Светодиод выключен</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLedON_2">
           <property name="text">
            <string>Включить светодиод</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLedOFF_2">
           <property name="text">
            <string>Выключить светодиод</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSendData">
           <property name="text">
            <string>Отправить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderPWM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelPWM</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
